sql/moz-fx-data-shared-prod/apple_ads_external/ios_app_campaign_stats_v1/query.sql

```diff
@@ -2,61 +2,63 @@ WITH daily_stats AS (
   SELECT
     date_day AS `date`,
     campaign_name AS campaign,
-    campaign_id AS campaign_id,
+    campaign_id,
     REGEXP_EXTRACT(
       campaign_name,
       r'Mozilla_(?:FF|Firefox)_ASA_(?:iOSGeoTest_)?([\w]{2,3})_.*'
     ) AS campaign_country_code,
     ad_group_name,
     ad_group_id,
-    DATE_TRUNC(date_day, WEEK) AS week_start,
-    FORMAT_DATE("%b", date_day) AS `month`,
+    -- DATE_TRUNC(date_day, WEEK) AS week_start, -- TODO: this doesn't appear to be used later on, should it be?
+    -- FORMAT_DATE("%b", date_day) AS `month`, -- TODO: this doesn't appear to be used later on, should it be?
     SUM(spend) AS spend,
     SUM(taps) AS clicks,
     SUM(impressions) AS impressions,
-    SUM(redownloads) AS redownloads,
-    SUM(total_downloads) AS total_downloads
+    SUM(redownloads) AS redownloads, -- TODO: this doesn't appear to be used later on, is this needed?
```

Comment:
@kwindau  appears to not be referenced downstream, any objections against just removing this code?

Reply:
Not sure, please check with @soGaussian

Reply:
Keep this one

Reply:
@soGaussian does that mean I should add this field to the final result set? Currently, it is not used outside of this CTE.

Reply:
Then clear it